tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -375,6 +374,66 @@ async function connectCommerceMock(mockHost, tokenData) {
   }
 }
 
+async function renewCommerceMockCertificate() {
+  const vs = await waitForVirtualService("mocks", "commerce-mock");
+  const mockHost = vs.spec.hosts[0];
+  const url = `https://${mockHost}/connection/renew`;
+  const params = {
+    headers: {
+      "Content-Type": "application/json"
+    },
+    timeout: 5000,
+  };
+
+  try {
+    await axios.post(url, {}, params);
+  } catch (err) {
+    throw convertAxiosError(err, "Error during renewing Commerce Mock certificate via Kyma connector service");
+  }
+}
+
+async function getCommerceMockCertFiles() {
+  let cert, key;
+
+  const vs = await waitForVirtualService("mocks", "commerce-mock");
+  const mockHost = vs.spec.hosts[0];
+  try {
+    cert = await axios.get(`https://${mockHost}/connection/cert`);
+    key = await axios.get(`https://${mockHost}/connection/key`);
+  } catch (err) {
+    throw convertAxiosError(err, "Error during obtaining Commerce Mock certificate private files");
+  }
+
+  return {cert, key}
+}
+
+async function revokeCommerceMockCertificate(){
+  let {cert, key} = await getCommerceMockCertFiles()
+   
+  const vs = await waitForVirtualService("mocks", "commerce-mock");
+  const mockHost = vs.spec.hosts[0];
+  const url = mockHost.replace(/(commerce.?)/,'');
+  const gateway = `https://gateway.${url}/v1/applications/certificates/revocations`;
+  const httpsAgent = new https.Agent({
+    //rejectUnauthorized: false, // curl -k
```

Comment:
Should it be there?

Reply:
Yes, it should be there, because I need to remove the unnecessary `commerce` prefix. And finally, create a new `httpAgent` with the required credentials.

Reply:
I mean the comment :D If the comment should be there, I suggest making it more verbose because we may end up not knowing what's that about